models/search_result.coffee

```diff
@@ -74,3 +93,70 @@ module.exports = class SearchResult extends Backbone.Model
       @set display_model: 'Booth'
     else
       @set location: "#{fair.href()}/browse#{@get('location')}"
+
+  status: ->
+    if @get('model') == 'partnershow'
+      if startTime = @get('start_at')
+        if endTime = @get('end_at')
+          if moment() > moment(endTime)
```

Comment:
I'm surprised this works, maybe what you're looking for is [isAfter](https://momentjs.com/docs/#/query/is-after/).